Gruntfile.js

```diff
@@ -334,9 +334,13 @@ module.exports = function(grunt) {
 			// Grab the relevant bits from the file contents
 			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
 
+			// Grab the filename
+			var name = svgFile.split( '.' );
+			name = name[0];
 
 			// Add transparent rectangle to each file
 			fileContent = fileContent.slice( 0, fileContent.indexOf('viewBox="0 0 24 24">') + 20 ) +
+						'<title>' + name + '</title>' +
```

Comment:
Maybe add this step to `svgmin` instead. `addsquare` is run last and appears to be used for sketch files?

Also there's lots of ways to do it but how about:
```javascript
var title = name.split( '-' ).splice( 1 ).map( function( item ) { 
    return item.charAt(0).toUpperCase() + item.slice(1); 
} ).join( ' ' );
```